packages/components/src/components/relative-datepicker/relative-datepicker.scss

```diff
@@ -0,0 +1,18 @@
+#dc-relative-datepicker {
+    width: 100%;
+    text-align: center;
+    padding-bottom: 16px;
+
+    span {
+        color: var(--purchase-main-2);
```

Comment:
You can use the sass mixin `@include typeface(--xsmall-center-normal-black)` and remove lines 8 to 10. Keep in mind the color css rule should occur after the aforementioned mixin.

Reply:
:+1: